lib/decidim/challenges/engine.rb

```diff
@@ -10,13 +10,18 @@ class Engine < ::Rails::Engine
       isolate_namespace Decidim::Challenges
 
       routes do
-        resources :challenges
         root to: "challenges#index"
+        resources :challenges, only: [:show] do end
```

Comment:
Why are you scoping challenges routes with only show ?

Reply:
correct, we also need the :index action
```suggestion
        resources :challenges, only: [:index, :show]
```
and remove the block as it is empty please